PhmarcyManagement/app/src/main/java/com/mahibul/phmarcymanagement/data/reposotory/due_coustomer/DataCustomer.kt

```diff
@@ -0,0 +1,6 @@
+package com.mahibul.phmarcymanagement.data.reposotory.due_coustomer
+
+data class DataCustomer(
+        val customer_name : String,
+        val customer_due : Int
```

Comment:
Class name should `Customer`

Variable name should be like: `name`, `dueAmount`.